orders/src/main/java/com/lambdaschool/orders/model/Customers.java

```diff
@@ -0,0 +1,190 @@
+package com.lambdaschool.orders.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "customers")
+public class Customers
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long custcode;
+
+    @Column(nullable = false)
+    private String custname;
+
+    private String custcity;
+    private String workingarea;
+    private String custcountry;
+    private String grade;
+    private double openingamt;
+    private double receiveamt;
+    private double paymentamt;
+    private double outstandingamt;
+    private String phone;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "agentcode",
+            nullable = false)
+    @JsonIgnoreProperties({"customers", "hibernateLazyInitializer"})
+    private Agents agent;
```

Comment:
Line 33 - Make sure to include the field name "orders" so that the agent object assigned to a customer will not print out its list of orders (which will include orders from other customers too).